tensorflow/lite/core/api/flatbuffer_conversions.cc

```diff
@@ -400,6 +400,7 @@ TfLiteStatus ParseOpData(const Operator* op, BuiltinOperator op_type,
       if (const auto* schema_params = op->builtin_options_as_AddOptions()) {
         params->activation =
             parse_activation(schema_params->fused_activation_function());
+        params->pot_scale_int16 = schema_params->pot_scale_int16();
```

Comment:
So this isn't actually used in the kernel implementation? It seems like the value is simply being recomputed, so why do we need this in the flatbuffer?

In which case, is it only used to populate the op version? In that case, I'm curious if there's any way to dynamically determine the op version without creating an arbitrary unused field in the flatbuffer? Can we inspect the scales directly when doing the versioning check?

Reply:
The first implementation of versining was like this:
https://github.com/tensorflow/tensorflow/pull/35985/commits/dd0d9e8f03d1fb1b887609fffb8ea5a86638c63e

Does this look correct ?
I addedo flatbuffer schema as it was requested after review of the commit above.